test/functional/system/2.2.delegates/1.same.account.same.username.js

```diff
@@ -28,6 +28,7 @@ describe('system test (type 2) - double delegate registrations', () => {
 
 	var i = 0;
 	var t = 0;
+	/* eslint-disable no-loop-func */
 	while (i < 30) {
 		describe('executing 30 times', () => {
```

Comment:
This should have a different description each time at least. Maybe we need an issue to revisit these tests?

Reply:
There is actually a eslint rule for identical title, so we can club these issues along with that. This kind of description issues present across the project, so its a good idea to have a separate issue for this.